src/libraries/Microsoft.CSharp/README.md

```diff
@@ -0,0 +1,8 @@
+# Microsoft.CSharp package
+
+We are not accepting feature contributions to Microsoft.CSharp.
+The package is effectively archived.
```

Comment:
Not sure if this is correct.  Archived implies no changes whatsoever.

Reply:
The phrase "effectively archived" is being applied to all of our code bases where we're only taking servicing level fixes. It's how we describe the old corefx, coreclr and core-setup repositories for example. These libraries fit into that theme.

Reply:
I see.  It's up to you what language you use, but it came across as hyperbole to me when the following statement seemed to contradict it.  To me, "effectively archived" makes sense for repositories where the code has moved somewhere else for the latest release, it makes less sense when the library is still building out of the codebase and you've just racheted up the bar: nothing has been archived it just has a higher bar.